src/web/src/components/BannerButtons.tsx

```diff
@@ -58,7 +58,7 @@ const LandingButtons = () => {
           >
             Sign out
           </Button>
-          <PostAvatar name={user.name} />
+          <PostAvatar name={user.name} img={user.avatarUrl} />
```

Comment:
Won't the img give problems if the avatarUrl is not defined here? If decoded.picture is the user's avatar url, we might want to use that instead

Reply:
The img will return first:
```
const PostAvatar = ({ name, img, blog = '' }: AvatarProps) => {
  const classes = useStyles();

  if (img) {
    return <Avatar className={classes.avatar} src={img} />;
  }

  if (name.length > 0) {
    const initials = name
   ```
   
   https://github.com/Seneca-CDOT/telescope/blob/baf9c91d25bcf96919951e10e4d9cb1caef81977/src/web/src/components/Posts/PostAvatar.tsx#L42

Reply:
The `<PostAvatar>` only renders if `img` exists, otherwise we do the initials.  The `picture` property from the JWT payload is stored in the `User` type as `avatarUrl`.